galter_subjects_utils/updater.py

```diff
@@ -22,78 +26,114 @@ def filter_ops_by_type(ops_data, _type):
     return [op for op in ops_data if op.get("type") == _type]
 
 
-def op_remove(subjects, op_data):
-    """Remove subject in-place.
+def find_idx_subject_dict(subjects, id_):
+    """Return (idx, subject_dict) of subject with `id_` in `subjects`.
+
+    Return (-1, {}) if not found.
+    """
+    return next(
+        ((i, s) for i, s in enumerate(subjects) if s.get("id") == id_),
+        (-1, {})
+    )
+
 
-    This is actually a rare op, because usually subjects are replaced.
+def op_replace(subjects, op_data):
```

Comment:
Just moving definition of `op_replace` above `op_remove` to mimic their order in `apply_op_data_change`